resources/hu/cemu.po

```diff
@@ -2565,7 +2575,7 @@ msgstr "Textúragyorsítótár"
 
 #: textureRelationWindow.cpp
 msgid "PhysAddr"
-msgstr "Fizikus Cím"
+msgstr "Fizikai cím"
```

Comment:
It is probably better to leave this untranslated like other strings that refer to Cemu internals (my fault for not catching this earlier).

Reply:
Also fixing this.